Makefile.vendor.mk

```diff
@@ -1,6 +1,6 @@
-VERSION = 3.0.0
+VERSION = 3.0.0-tp.1
```

Comment:
Shouldn't it be 3.0.0.tp.1?

Reply:
Both should be fine according to https://semver.org/. I can change it to `3.0.0.tp.1`, we can choose here. The place where it's strict is the image version because of OSBS - https://gitlab.cee.redhat.com/istio/cpaas/containers-midstream/-/commit/39fc86a6bfaf31e8814a79a4400bb111f8de7256